cmd/testmode/testrun.go

```diff
@@ -43,6 +78,125 @@ func testStatusToString(status checks.TestStatus) string {
 	}
 }
 
+func cmdMagicToString(magic memd.CmdMagic) string {
+	switch magic {
+	case memd.CmdMagicReq:
+		return "request"
+	case memd.CmdMagicRes:
+		return "response"
+	default:
+		return fmt.Sprintf("unk:%d", magic)
+	}
+}
+
+func durabilityLevelToString(level memd.DurabilityLevel) string {
```

Comment:
I wonder if we should move some of these into the memd package at some point, and hang them off the actual types. This function for instance exists elsewhere.